docs/src/modules/sandbox/Dependencies.ts

```diff
@@ -51,8 +51,11 @@ export default function SandboxDependencies(demo: Demo, options?: { commitRef?:
       commitRef === undefined ||
       process.env.SOURCE_CODE_REPO !== 'https://github.com/mui/material-ui'
     ) {
-      // #default-branch-switch
-      return 'latest';
+      if (['joy', 'base'].includes(packageName)) {
+        return 'latest';
+      }
+      // #npm-tag-reference
+      return 'latest-v6';
```

Comment:
What's the motivation for using tag for the latest v6? Can't we just use "^6.0.0"? I understand the tag is useful for switching between latest and next, but here I don't see the value.

Reply:
This makes sense, let's use `^6.0.0`